Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: install install-dev install-pre-commit test style docs docs-serve
+
+install:
+	pip install sqlglot
+
+install-dev:
+	pip install -e ".[dev]"
+
+install-pre-commit:
+	pre-commit install
+
+test:
+	python -m unittest
+
+style:
+	pre-commit run --all-files
+
```

Comment:
There should be one command that does both `make test` and `make style`

Reply:
What about `make checks`?

Reply:
Done.